proknow-sdk-test/TestHelper.cs

```diff
@@ -85,6 +87,73 @@ public static async Task AssemblyCleanup()
             }
 
             return patientItem;
+        }
+
+        /// <summary>
+        /// Creates an array of patients asynchronously in workspace {testClassName} with MRN {testNumber}-i-Mrn and name {testNumber}-i-Name
+        /// </summary>
+        /// <param name="testClassName">The test class name</param>
+        /// <param name="testNumber">The test number</param>
+        /// <param name="numPatients">The number of patients to create (informs the range of values of "i" for MRN and name)</param>
+        /// <returns></returns>
+        public static async Task<IList<PatientItem>> CreateMultiPatientAsync(string testClassName, int testNumber, int numPatients)
+        {
+            // Find the workspace for this test
+            var workspaceName = $"SDK-{testClassName}-{testNumber}";
+            var workspaceItem = await _proKnow.Workspaces.ResolveByNameAsync(workspaceName);
+
+            //+++++++++++++++++++++++++++++++
+            // Here are two option for how to handle sending the patient creation request concurrently
+            //    without screwing up the communication between this test and ProKnow (WSL issue).
+            // Pick your favorite and I'll clean up the code accordingly
+            //+++++++++++++++++++++++++++++++
+
+            //// Create the patients
+            //var patientCreateTasks = new List<Task<PatientItem>>();
+            //for (int i = 0; i < numPatients; i++)
+            //{
+            //    var mrn = $"{testNumber}-{i}-Mrn";
+            //    var name = $"{testNumber}-{i}-Name";
+            //    if (i % 500 == 0)
+            //    {
+            //        await Task.Delay(1000);

```

Comment:
I think both ways looks good. We can even make it to delay for every 250 patients to be more safe.
Let me know what you think @kyleburnett

Reply:
I'm okay with either as well. Whichever you think would be faster as long as both are reliable.